src/applications/personalization/profile-2/tests/e2e/profile.not-in-mpi-error.cypress.spec.js

```diff
@@ -0,0 +1,95 @@
+import { PROFILE_PATHS } from '../../constants';
```

Comment:
This test is nearly identical to the existing `mpi-error` test. The difference is we are using a different mock user and are checking for a different error message.